src/templates/pt/base.html

```diff
@@ -20,7 +20,7 @@
 
 {% block mission %}
 <p>
-Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. É composto por {{ supported_chapters | length }} capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
+Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. A edição de {{ year }} é composta por {{ supported_chapters | length }} capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
```

Comment:
Reading becomes more fluid using "comunidade web" instead of "comunidade da web".

Reply:
```suggestion
Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. A edição de {{ year }} é composta por {{ supported_chapters | length }} capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
```